lighthouse-cli/index.js

```diff
@@ -62,6 +62,7 @@ const cli = yargs
     'list-all-audits': 'Prints a list of all available audits and exits',
     'list-trace-categories': 'Prints a list of all required trace categories and exits',
     'config-path': 'The path to the config JSON.',
+    'launch-chrome': 'Launch chrome automatically when accessing port 9222 fails',
```

Comment:
Your comments in the PR say `--no-launch-chrome` but the code is just `--launch-chrome`. It looks like you're using http://yargs.js.org/docs/#parsing-tricks-negate-fields to do that? whoa, TIL! very cool.

However, I prefer only really exposing the user to `--no-launch-chrome` or `--no-autolaunch` in `--help` and such. Is that possible?